### ISSUE TITLE
[Pomodoro] Random Quote Generating Problem

### ISSUE THREAD

**Issue by kayoooon1** [raycast/extensions#13417]
### Extension

https://raycast.com/asubbotin/pomodoro

### Raycast Version

1.78.1

#### Error:
```
FetchError: request to https://api.quotable.io/quotes/random?limit=1 failed, reason: getaddrinfo ENOTFOUND api.quotable.io
    at ClientRequest.<anonymous> (/Users/name/.config/raycast/extensions/da2ac998-edd2-49a4-8614-06b57f30f619/pomodoro-control-timer.js:8:311184)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

### Steps To Reproduce

I set the timer to show a quote after I finish my interval, but it doesn't show any quote and prints error to fetch a quote.

**Comment by mikikiv** [comment]
Looks like the api from quotable is down. Nothing we can do about that here unless there is another service we can use
https://github.com/lukePeavey/quotable/issues

**Comment by j3lte** [comment]
Fixed in #13724 

@raycastbot close this issue
